sdcm/utils/adaptive_timeouts/__init__.py

```
@@ -143,6 +143,39 @@ def _get_service_level_propagation_timeout(
     return timeout, stats
 
 
+def _get_compaction_timeout(
+    node_info_service: NodeLoadInfoService, timeout: int | float = None
```

Ruff (0.15.20)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)